migrations/1754109724.sh

```diff
@@ -0,0 +1,5 @@
+echo "Add hyprsunset blue light filter"
+yay -S --noconfirm --needed hyprsunset
```

Comment:
Wrap this in a `! command -v hyprsunset` test, like some of the other migrations.